### PR TITLE
ROC-3575 Enable CCD by default locally

### PR DESCRIPTION
**Pull request by timja-kainos**
### JIRA link ###
https://tools.hmcts.net/jira/browse/ROC-3575

### Change description ###
Enable CCD by default locally

Note: case-management-web doesn't work yet, and only the gateway is being published currently, will probably manually push one version while waiting for the last automation to be done

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: bin/ccd/import-definition.sh [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 if [ -z "$1" ]
   then
-    echo "Usage: ./import-definition.sh path_to_definition [userToken] [serviceToken]"
+    echo "Usage: ./import-definition.sh path_to_definition [userToken]"
```
Comment (geek-soft) [inline bin/ccd/import-definition.sh]: Please update DOC (first few lines) as well.

File: docker/ccd-definition-import/Dockerfile [hunk 2]
```diff
@@ -1,20 +1,14 @@
...
 
 ## Add the wait script to the image
 ADD https://github.com/ufoscout/docker-compose-wait/releases/download/2.0.0/wait /wait
 RUN ["chmod", "+x", "/wait"]
 
-RUN ["chmod", "+x", "/create-importer-user.sh"]
-RUN ["chmod", "+x", "/add-ccd-role.sh"]
-RUN ["chmod", "+x", "/import-definition.sh"]
-RUN ["chmod", "+x", "/upload-definition.sh"]
+RUN chmod +x /scripts/*.sh
```
Comment (geek-soft) [inline docker/ccd-definition-import/Dockerfile]: Please move `COPY scripts /scripts` close before that line. This way any change to scripts will not trigger download of `https://github.com/ufoscout/docker-compose-wait/releases/download/2.0.0/wait`.

File: docker/ccd-definition-import/scripts/add-ccd-role.sh [hunk 3]
```diff
@@ -13,11 +13,10 @@
 role=$1
 classification=$2
 userToken=$3
-serviceToken=$4
 
 if [ -z "$role" ]
   then
-    echo "Usage: ./add-ccd-role.sh role [classification] [userToken] [serviceToken]"
+    echo "Usage: ./add-ccd-role.sh role [classification] [userToken]"
```
Comment (geek-soft) [inline docker/ccd-definition-import/scripts/add-ccd-role.sh]: Please update DOC here as well.

File: docker-compose.yml [hunk 4]
```diff
@@ -222,10 +222,10 @@ services:
         - http_proxy
         - https_proxy
         - no_proxy=idam-api,service-auth-provider-api,pdf-service-api,ccd-data-store-api
-        - IDAM_ANONYMOUS_CASEWORKER_USERNAME=civilmoneyclaims+anonymouscitizen@gmail.com
-        - IDAM_ANONYMOUS_CASEWORKER_PASSWORD=Password12
-        - IDAM_SYSTEM_UPDATE_USER_USERNAME=civilmoneyclaims+systemupdate@gmail.com
-        - IDAM_SYSTEM_UPDATE_USER_PASSWORD=Password12
+        - IDAM_ANONYMOUS-CASEWORKER_USERNAME=civilmoneyclaims+anonymouscitizen@gmail.com
```
Comment (geek-soft) [inline docker-compose.yml]: Is that change needed? Wouldn't old property with underscores only work as well?

File: docker/ccd-definition-import/scripts/upload-definition.sh [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+# create user with ccd-import role
+/scripts/create-importer-user.sh
+
+IMPORTER_USERNAME=ccd-importer@server.net
+IMPORTER_PASSWORD=Password12
+IDAM_URI=http://idam-api:8080
+REDIRECT_URI=http://localhost:3000/receiver
+CLIENT_SECRET=123456
+
+code=$(curl --silent -u "${IMPORTER_USERNAME}:${IMPORTER_PASSWORD}" -XPOST "${IDAM_URI}/oauth2/authorize?redirect_uri=${REDIRECT_URI}&response_type=code&client_id=cmc_citizen" | jq -r .code)
```
Comment (geek-soft) [inline docker/ccd-definition-import/scripts/upload-definition.sh]: Please extract authentication logic to function that can be imported here and in the other script that uploads definition to non local env. I am worried that we have that logic already in 2 other places such as two frontends and backend. Having two more copies to maintain in form of scripts just puts extra maintenance overhead on us.

File: docker-compose.yml (hunk 4 above)
Comment (timja-kainos) [reply docker-compose.yml]: no it doesn't work since spring boot 2 upgrade apparently,
